tools/header_guard_check/pubspec.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This package should be listed in https://github.com/flutter/engine/blob/main/tools/pub_get_offline.py#L22

Reply:
Ack. I was just getting feedback on whether to continue this work mostly :)